pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = ["setuptools",
-            "Cython>=0.29",
+            "Cython>=0.29,<3",
```

Comment:
why? Cython 3 appears to work with cypari2.

Reply:
It should. I was wondering were these weird compilation errors came from. I will remove the commit.

Reply:
Already done